content/post/110-go-ipfs-0-7-0.md

```diff
@@ -0,0 +1,80 @@
+
+Many commands will accept a `--ipns-base` option that allows changing command outputs to use a particular encoding (i.e.  base58btc multihash, or CIDv1 encoded in any supported base)
+
+## 📮  Multiaddresses now accept PeerIDs encoded as CIDv1
+
+In preparation for eventually changing the default PeerID representation multiaddresses can now contain strings like `/p2p/k51xyz...` in addition to the default `/p2p/Qmabc...`. There is a corresponding `--peerid-base` option to many functions that output peerIDs.
+
+## 🧮 `dag stat`
+
+Initial support has been added for the `ipfs dag stat` command. Running this command will traverse the DAG for the given root CID and report statistics. By default, progress will be shown as the DAG is traversed. Supported statistics currently include DAG size and number of blocks.
+
+```bash
+ipfs dag stat bafybeihpetclqvwb4qnmumvcn7nh4pxrtugrlpw4jgjpqicdxsv7opdm6e # the IPFS webui
+Size: 30362191, NumBlocks: 346
+```
+
+## 🔌 Plugin build changes
+
+We have changed the build flags used by the official binary distributions on dist.ipfs.io (or `/ipns/dist.ipfs.io`) to use the simpler and more reliable `-trimpath` flag instead of the more complicated and brittle `-asmflags=all=-trimpath="$(GOPATH)" -gcflags=all=-trimpath="$(GOPATH)"` flags, however the build flags used by default in go-ipfs remain the same.
+
+The scripts in https://github.com/ipfs/go-ipfs-example-plugin have been updated to reflect this change. This is a breaking change to how people have been building plugins against the dist.ipfs.io binary of go-ipfs and plugins should update their build processes accordingly see https://github.com/ipfs/go-ipfs-example-plugin/pull/9 for details.
+
+## The Changelog
+
+For a full list of updates included in this release you can review the Changelog at https://github.com/ipfs/go-ipfs/blob/v0.7.0/CHANGELOG.md#v070-2020-09-22.
+
+## Thank you contributors!
+
+A huge thank you to [everyone who contributed](https://github.com/ipfs/go-ipfs/blob/v0.7.0/CHANGELOG.md#contributors) patches and improvements in this release, all **53** of you! We couldn’t have made this happen without your help and feedback. ❤
+
+## Install, upgrade, and join us!
+
+You can get started by [installing go-ipfs](https://dist.ipfs.io/#go-ipfs) or [upgrading to go-ipfs 0.7](https://docs.ipfs.io/recent-releases/go-ipfs-0-7/update-procedure).
```

Comment:
https://docs.ipfs.io/recent-releases/go-ipfs-0-7/update-procedure is missing atm –  @johnnymatthews  is there a PR for adding it?

Reply:
There isn't a page for `recent-releases/go-ipfs-0-7` yet. I'll make it now though. Gonna steal heavily from this blog post. Hope that's ok.

Reply:
I'll create a PR to the ipfs repo to include the update process docs in the release check list so we can make sure we start that update process earlier.

Reply:
Sorry for the delay. [docs.ipfs.io/recent-releases/go-ipfs-0-7-0](https://docs.ipfs.io/recent-releases/go-ipfs-0-7/) is now live-ish. I just pushed, so it might take about an hour or so to propagate through the interwebs.